### PR TITLE
Upgrading to scap 1.3

### PR DESCRIPTION
**Pull request by ihenyene**
Building the 1.3 content 
https://trello.com/c/hM3A1a2W/2267-scap-formula-update-scap-content-with-complianceascode-content-v0166

### CODE REVIEW

File: scripts/build_ssg_content.sh [hunk 1]
```diff
@@ -51,14 +51,14 @@ cd build
...
 # Build RHEL first to generate dependencies for CentOS and Oracle Linux
 make -j4 "${MAKE_TARGETS_RHEL[@]}"
 make -j4 "${MAKE_TARGETS_OTHERS[@]}"
 
-find . -type f -name '*ds-1.2*' -exec rename ds-1.2.xml ds.xml '{}' \;
-find . -type f -name '*xccdf-1.2*' -exec rename xccdf-1.2.xml xccdf.xml '{}' \;
+find . -type f -name '*ds-1.2*' -exec rename ds-1.3.xml ds.xml '{}' \;
+find . -type f -name '*xccdf-1.2*' -exec rename xccdf-1.3.xml xccdf.xml '{}' \;
 
 cp *-ds.xml *-xccdf.xml *-oval.xml *-cpe-dictionary.xml "$DIST_DIR"
```
Comment (lorengordon) [inline scripts/build_ssg_content.sh:63]: Ok, reviewing the built files, I think we can skip the rename step entirely, and just grab all the `ssg-*.xml` files.
```suggestion

cp ssg-*.xml "$DIST_DIR"
```

File: build/content
Comment (eemperor) [inline build/content:1]: Also remove this folder.  It's been added as a submodule.

File: appveyor.yml [hunk 2]
```diff
@@ -5,15 +5,17 @@ environment:
   global:
     SALT_FILEROOT: '%APPVEYOR_BUILD_FOLDER%'
     SALT_STATE: scap.scan
-    SALT_URL: https://repo.saltproject.io/windows/Salt-Minion-3004.2-1-Py3-AMD64-Setup.exe
+    SALT_URL:  'https://repo.saltproject.io/windows/Salt-Minion-3005.1-5-Py3-AMD64-Setup.exe'
```
Comment (eemperor) [inline appveyor.yml:8]: The single quotes aren't necessary.

File: appveyor.yml [hunk 3]
```diff
@@ -5,15 +5,17 @@ environment:
...
 init:
-  - ps: echo $env:PILLAR_HOME
+  - ps: |
+      echo $env:PILLAR_HOME
 install:
   - ps: |
       $null = `
+        [Net.ServicePointManager]::SecurityProtocol = "Tls12, Tls13"
         (new-object net.webclient).DownloadFile(${env:SALT_URL}, `
         "${env:temp}\salt-minion.exe")
```
Comment (eemperor) [inline appveyor.yml:20]: Try something like this:

```suggestion
  - ps: [Net.ServicePointManager]::SecurityProtocol = "Tls12, Tls13"
  - ps: |
      $null = `
        (new-object net.webclient).DownloadFile(${env:SALT_URL}, `
        "${env:temp}\salt-minion.exe")
```

File: README.md [hunk 4]
```diff
@@ -83,7 +83,7 @@ that represent the security guides to use to scan the system. The paths are
 relative to the `content.source` pillar key. The pattern will be matched by
 globbing a `*` to the end of the value.
 
-## Configuration
+### Configuration
```
Comment (eemperor) [inline README.md:86]: Remove the additional `#`

```suggestion
## Configuration
```

File: scap/content/guides/openscap/ssg-centos7-ds-1.2.xml
Comment (lorengordon) [inline scap/content/guides/openscap/ssg-centos7-ds-1.2.xml:1]: Can remove this file now

File: scap/content/guides/openscap/ssg-centos8-ds-1.2.xml
Comment (lorengordon) [inline scap/content/guides/openscap/ssg-centos8-ds-1.2.xml:1]: Can remove this file now

File: scap/content/guides/openscap/ssg-ol8-ds-1.2.xml
Comment (lorengordon) [inline scap/content/guides/openscap/ssg-ol8-ds-1.2.xml:1]: Can remove this file now

File: scap/content/guides/openscap/ssg-rhel7-ds-1.2.xml
Comment (lorengordon) [inline scap/content/guides/openscap/ssg-rhel7-ds-1.2.xml:1]: Can remove this file now

File: scap/content/guides/openscap/ssg-rhel8-ds-1.2.xml
Comment (lorengordon) [inline scap/content/guides/openscap/ssg-rhel8-ds-1.2.xml:1]: Can remove this file now

File: scap/content/guides/openscap/ssg-rhel8-ocil.xml
Comment (lorengordon) [inline scap/content/guides/openscap/ssg-rhel8-ocil.xml:1]: Can remove this file now

File: scripts/build_ssg_content.sh [hunk 5]
```diff
@@ -51,16 +51,14 @@ cd build
 # Need to build content to support SCAP 1.2
 # more info here: https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer_guide.adoc#331-building-compliant-scap-12-content
```
Comment (lorengordon) [inline scripts/build_ssg_content.sh:52]: Let's go ahead and update this comment.

```suggestion
# Build SCAP 1.3 content
# more info here: https://complianceascode.readthedocs.io/en/latest/manual/developer/02_building_complianceascode.html#building
```

File: scripts/build_ssg_content.sh [hunk 6]
```diff
@@ -48,19 +48,17 @@ echo
 cd build
 
 # OpenSCAP content default supports SCAP 1.3. Salt installs SCAP 1.2
```
Comment (lorengordon) [inline scripts/build_ssg_content.sh:50]: Note the new comment replaces this line also.

**Review by lorengordon — APPROVED** [review]

**Review by eemperor — APPROVED** [review]

**Pull request merged**